openml/config.py

```diff
@@ -157,6 +158,9 @@ def start_using_configuration_for_example(cls):
         # Test server key for examples
         server = cls._test_server
         apikey = cls._test_apikey
+        warnings.warn(
+            "Switching to the test server {} may result in inconsistent behaviour!".format(server)
```

Comment:
```suggestion
            "Switching to the test server {} to not upload from an example to the live server.
            Using the test server may result in reduced performance of the API!".format(server)
```

Reply:
I don't think the "to not upload from an example to the live server." part of the warning emitted by code is necessary, because it assumes _why_ the user called the function. So personally I would leave that out, but it's ok with me either way.

I think the new warning text in the examples is great! Though it might be a good idea to look into a way to use a token instead (e.g.`REPLACE_WITH_TEST_SERVER_WARNING`) which gets inserted when converted to a web page to keep the copied text central. I'm not sure if there's a good/standard way to do this, so if you want to keep the repeated warning texts that's fine with me too (we can always come back to it later).

Reply:
I wouldn't know how to extract the warning text to a central place, I don't think that feature exists. I also changed the text, so hopefully it's better now.

Reply:
Wait, I think I looked into the wrong docs, this would be i sphinx/rst feature, not a sphinx gallery feature: https://stackoverflow.com/a/2922004